install/source/index.markdown

```diff
@@ -98,6 +98,62 @@ Note, every command to ``catkin config --blacklist`` will override the previous
          moveit_planners_chomp \
          chomp_motion_planner
 
+## Optional: Install a Compiler Cache
+
+Even if you decide to not build *all* of MoveIt, building it from source can still take quite some time.
+Because of this, it can be beneficial to use a tool called a *compiler cache* to avoid recompiling files that have not been changed between builds but which with a regular build (without such a cache) would still be processed by the compiler.
+This situation is especially common when switching between different branches while developing and when `clean`ing a Catkin workspace and then rebuilding it.
+
+[ccache](https://ccache.dev) provides such a compiler cache and installing it can reduce the time needed to rebuild MoveIt (and other packages) from many minutes to mere seconds.
```

Comment:
```suggestion
[ccache](https://ccache.dev) provides such a compiler cache and using it can reduce the time needed to rebuild MoveIt (and other packages) from many minutes to mere seconds.
```